doc/Requirements Specification/Parts/Purpose.lyx

```diff
@@ -670,7 +670,111 @@ name "/OP10/"
 
 \end_inset
 
- Beagle analyses given source code for further dynamic behavioural attributes.
+ Beagle analyses given source code for further dynamic behavioural attributes
+ like the number of loop executions or the probability for branches in 
+\begin_inset ERT
+status open
+
+\begin_layout Plain Layout
+
+
+\backslash
+lstinline{if-else}
+\end_layout
+
+\end_inset
+
+ or 
+\begin_inset ERT
+status open
+
+\begin_layout Plain Layout
+
+
+\backslash
+lstinline{switch-case}
+\end_layout
+
+\end_inset
+
+ constructs.
```

Comment:
couldn’t you use the defined Term “SEFF condition” instead of “`if-else` or `switch` constructs”?